homeassistant/components/firmata/config_flow.py

```diff
@@ -0,0 +1,39 @@
+"""Config flow to configure firmata component."""
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME
+from homeassistant.core import callback
+
+from .const import CONF_SERIAL_PORT, DOMAIN
+
+
+@callback
+def configured_boards(hass):
+    """Return a set of all configured boards."""
+    return {
+        entry.data[CONF_NAME]: entry
+        for entry in hass.config_entries.async_entries(DOMAIN)
+    }
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class FirmataFlowHandler(config_entries.ConfigFlow):
```

Comment:
Pass the domain as keyword argument domain and remove the decorator.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#defining-your-config-flow